### PR TITLE
Update spring integration migration-guide

### PR DESCRIPTION
**Pull request by hui1110**
Fix the following issues:
1. [The table is not displayed completely.](https://microsoft.github.io/spring-cloud-azure/current/reference/html/appendix.html#_from_azure_spring_cloud_starter_servicebus_to_spring_cloud_azure_starter_integration_servicebus)
2. last row of the table，[attribute error.](https://microsoft.github.io/spring-cloud-azure/current/reference/html/appendix.html#_from_azure_spring_boot_starter_active_directory_b2c_to_spring_cloud_azure_starter_active_directory_b2c)

### CODE REVIEW

File: docs/src/main/asciidoc/_migration-guide-for-4.0.adoc [hunk 1]
```diff
@@ -769,7 +769,7 @@ public class Demo {
 |com.azure.spring.integration.core.api.reactor.Checkpointer |com.azure.spring.messaging.checkpoint.Checkpointer
 |com.azure.spring.integration.core.api.CheckpointConfig |com.azure.spring.messaging.checkpoint.CheckpointConfig
 |com.azure.spring.integration.core.api.CheckpointMode |com.azure.spring.messaging.checkpoint.CheckpointMode
-|com.azure.spring.integration.core.api.reactor.DefaultMessageHandler |com.azure.spring.integration.handler.DefaultMessageHandler
+|com.azure.spring.integration.core.api.reactor.DefaultMessageHandler |com.azure.spring.integration.implementation.handler.DefaultMessageHandler
```
Comment (saragluna) [inline docs/src/main/asciidoc/_migration-guide-for-4.0.adoc:772]: I will revert this change in the source code.

Comment (saragluna) [reply docs/src/main/asciidoc/_migration-guide-for-4.0.adoc:772]: So please revert this too.

**Review by saragluna — CHANGES_REQUESTED** [review]

**Review by chenrujun — APPROVED** [review]
LGTM.

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**